index.html

```diff
@@ -4,13 +4,152 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Global Enviromental Forum</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
-  <link src="style.css" ref="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/db2c8620f4.js" crossorigin="anonymous"></script>
+  
 </head>
 <body>
-  <script src="index.js"></script>    
+  <header id="header-container">
+    <nav class="nav-bar">
+      <div class="mobile"><img class="hamburguer-menu" src="./Images/Hamburguer_Menu.png"/></div>
+      <a href="index.html"><img class="forum-logo" src="./Images/Forum Logo.png"/></a>
+      <ul class="navbar-menu-list desktop">
+        <li><a class="nav-bar-menu-li" href="about.html">About</a></li>
+        <li><a class="nav-bar-menu-li" href="#program">Program</a></li>
+        <li><a class="nav-bar-menu-li"href="#">Join</a></li>
+        <li><a class="nav-bar-menu-li">Sponsor</a></li>
+        <li><a class="nav-bar-menu-li" href="#">News</a></li>
+        <li><a class="gef-campaing" href="#">GEF Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
- [ ] Kindly include the top black bar for both pages as present in the design guidelines (desktop) ✌
- [ ] Also, please remove the logo from the mobile section. Although it is looking nice, but it is not present in the design guidelines 😥
- [ ] Please note that the hamburger icon has a transparent background. Also, can you please add a bit of padding to hamburger also? 😅⛏

Your design | Given design
---| ---|
![image](https://github.com/HFG43/Capstone1/assets/106119105/714e31b2-2b59-4b1d-82fd-d6c46280ed17) | ![image](https://github.com/HFG43/Capstone1/assets/106119105/ca55476c-b022-460e-b415-cd72c6872852)